os/pm/pm_procfs.c

```diff
@@ -148,6 +148,8 @@ static int power_stat(const char *relpath, FAR struct stat *buf);
 #define POWER_LEVEL_3   3
 #define POWER_LEVEL_4   4
 
+static const char *pm_state_name[PM_COUNT] = {"PM_NORMAL", "PM_IDLE", "PM_STANDBY", "PM_SLEEP"};
```

Comment:
will you remove duplicate?

Reply:
`	pmdbg(" BOARD STATE | PM STATE |          TIME          \n");
	pmdbg("-------------|----------|------------------------\n");
	pmdbg(" %11s | %8s | %10dms (%6.2f%%) \n", "WAKEUP", "NORMAL", TICK2MSEC(g_pm_metrics->state_metrics.state_accum_ticks[0]),
		  ((double)g_pm_metrics->state_metrics.state_accum_ticks[0]) * 100.0 / total_time);
	pmdbg(" %11s | %8s | %10dms (%6.2f%%) \n", "", "IDLE", TICK2MSEC(g_pm_metrics->state_metrics.state_accum_ticks[1]),
		  ((double)g_pm_metrics->state_metrics.state_accum_ticks[1]) * 100.0 / total_time);
	pmdbg(" %11s | %8s | %10dms (%6.2f%%) \n", "", "STANDBY", TICK2MSEC(g_pm_metrics->state_metrics.state_accum_ticks[2]),
		  ((double)g_pm_metrics->state_metrics.state_accum_ticks[2]) * 100.0 / total_time);
	pmdbg("-------------|----------|------------------------\n");
	pmdbg(" %11s | %8s | %10dms (%6.2f%%) \n", "SLEEP", "SLEEP", TICK2MSEC(g_pm_metrics->board_sleep_ticks),
		  ((double)g_pm_metrics->board_sleep_ticks) * 100.0 / total_time);`

Reply:
C don't support enum of strings, so we need to maintain separate string array for PM STATES name.

Reply:
Let's move this to pm_init and using extern

Reply:
@gSahitya-samsung I heard he suggested to use this array to print `"NORMAL", "IDLE", "STANDBY", and "SLEEP"` in the pm_metrics.c. You print them with hardcoded strings there.
Then, need to remove "PM_" prefix.

Reply:
Done